### ISSUE TITLE
Skyrim patcher

### ISSUE THREAD

**Issue by Ishikudeska** [Ishikudeska/Nolvus-Installer-V2#5]

**Comment by TheAndreLuiz** [comment]
I think we must patch AE cause of the other CC mods...

**Comment by Ishikudeska** [comment]
Yes, we can patch AE only. I think it'd be best to do the latest version only. Just so people don't have to try and patch it themselves to make it work with the installer

**Comment by Ishikudeska** [comment]
Considering using Vektor's skyrim patcher.
